dotCMS/src/main/java/com/dotcms/integritycheckers/IntegrityUtil.java

```diff
@@ -598,19 +599,19 @@ public void flushAllCache() {
 	 * which indicates what records <b>MUST</b> be changed in the specified end
 	 * point.
 	 * 
-	 * @param endpointId
+	 * @param key
 	 *            - The ID of the end point where the data will be fixed.
 	 * @param type
 	 *            - The type of object (Content Page, Folder, Content Type,
 	 *            etc.) that will be fixed.
 	 * @throws Exception
 	 *             An error occurred during the integrity fix process.
 	 */
-    public void generateDataToFixTable(String endpointId, IntegrityType type) throws Exception {
+    public void generateDataToFixTable(String key, IntegrityType type) throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6190239)